libs/SalesforceHybridSDK/SalesforceHybridSDK/Classes/SFHybridViewController.m

```diff
@@ -154,13 +160,20 @@ - (id)initWithConfig:(SFHybridViewConfig *)viewConfig
         _hybridViewConfig = (viewConfig == nil ? [SFHybridViewConfig fromDefaultConfigFile] : viewConfig);
         NSAssert(_hybridViewConfig != nil, @"_hybridViewConfig was not properly initialized. See output log for errors.");
         self.startPage = _hybridViewConfig.startPage;
-
-        // Setting our WKWebView based plugin as the engine to be used.
-        [self.settings setCordovaSetting:@"SFWKWebViewEngine" forKey:@"CordovaWebViewEngine"];
     }
     return self;
 }
 
+- (UIView *)newCordovaViewWithFrame:(CGRect)bounds
+{
+
+    // Setting our WKWebView based plugin as the engine to be used.
+    [self.settings setCordovaSetting:@"SFWKWebViewEngine" forKey:@"CordovaWebViewEngine"];
+    UIView *view = [super newCordovaViewWithFrame:bounds];
+    self.navWebViewDelegate = [[SFWKWebViewNavigationDelegate alloc] initWithEnginePlugin:self.webViewEngine];
```

Comment:
Now, this one is interesting. In the new model, you can only have 1 navigation delegate, which means, it's either your view controller (in our case), or the stock navigation delegate. So, we need to invoke default behavior as well (the equivalent of ```super``` call previously). The reason it's done here is because we need to ensure our engine is initialized before we do this.